backend/models/test.model.ts

```diff
@@ -137,12 +138,12 @@ const TestSchema: Schema = new Schema(
     assessmentType: {
       type: String,
       required: true,
-      enum: Object.keys(AssessmentType),
```

Comment:
is this change necessary? i believe it should work with the enum

Reply:
no it's not, I was just trying things - I can change it back